peps/pep-0763.rst

```diff
@@ -264,6 +269,115 @@ usage, this PEP identifies several limitations to this approach:
    PyPI is not straightforward, and there is limited possibility to gather a
    project's download statistics from mirrors or other distributions systems.
 
+.. _pep763-appendix-a:
+
+Appendix A: Precedent in other ecosystems
+=========================================
+
+The following is a table of support for deletion in different packaging
+ecosystems. An ecosystem is considered to **not** support deletion
+if it restrict's a user's ability to perform deletions in a manner similar
+to this PEP.
+
+An earlier version of this table, showing only deletion, was
+compiled by Donald Stufft and others on the Python discussion forum in
+`July 2022 <https://discuss.python.org/t/17227/58>`__.
```

Comment:
This is Adrian's post. Did you mean Donald's with the table?

```suggestion
`July 2022 <https://discuss.python.org/t/17227/59>`__.
```

Reply:
Yep. That's what I get for blindly copying the fragment from my browser.